pkg/controller/controller_utils.go

```diff
@@ -466,26 +466,26 @@ func SyncAllPodsWithStore(kubeClient clientset.Interface, store cache.Store) {
 	return
 }
 
-// ControllersByCreationTimestamp sorts a list of ReplicationControllers by creation timestamp, using their names as a tie breaker.
```

Comment:
Leave the rc related type around for other callers. It should be removed until all rc related code is removed.